invenio_opendata/testsuite/data/cms/cms-tools-dimuon-spectrum-2010.xml

```diff
@@ -0,0 +1,69 @@
+<collection xmlns="http://www.loc.gov/MARC21/slim">
+<record>
+<controlfield tag="001">FIXME</controlfield>
+<controlfield tag="005">FIXME</controlfield>
+<datafield tag="024" ind1="7" ind2=" ">
+<subfield code="2">DOI</subfield>
+<subfield code="a">FIXME</subfield>
```

Comment:
For small records, you can mint DOI directly.  (I'll take care of batch for MC datasets etc.)